tests/AbstractHackTest.hack

```diff
@@ -190,19 +190,19 @@ abstract class AbstractHackTest extends Facebook\HackTest\HackTest {
     expect($type?->getGenericTypes())->toBeEmpty();
 
     $type = $this->getFunction('returns_generic')->getReturnType();
-    expect($type?->getTypeName())->toBeSame('Vector');
+    expect($type?->getTypeName())->toBeSame('HH\\Vector');
```

Comment:
perhaps a Vector::class instead?

Reply:
I think the string is better,

- provides more documentation value to someone reading the test
- we'd probably want the test to start failing if the namespace of `Vector` changes, since that would be a very unexpected change that might need investigation

Reply:
Maybe test both?

Reply:
Good idea -- I'll update it.